Source/Core/VideoCommon/VertexShaderGen.h

```diff
@@ -38,6 +38,7 @@ struct vertex_shader_uid_data
   u32 numColorChans : 2;
   u32 dualTexTrans_enabled : 1;
   u32 pixel_lighting : 1;
+  u32 vertex_rounding : 1;
```

Comment:
You need to preserve the bit alignment,

Put this below ssaa, and subtract 1 from pad.